### DISCUSSION TITLE
milestone 3-2

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#346] [General]
Scrum meetings and issue verifications

**Comment by DavidCuevas3** [comment]
Started brainstorming and creating the first task for the first sprint of this milestone.

**Comment by nsfrontera** [comment]
Working on completing my task for the sprint

**Comment by Saul-Caban-UPRM** [comment]
Added the new tiles assets #325; waiting for the approval, added task to the board.

**Comment by fernandopizarro1** [comment]
Closed issue #283 and merged with main.

**Comment by ianramospr** [comment]
Worked on my first issue for this milestone. Currently waiting for review.

**Comment by AngelCIICMorales** [comment]
Reviewed tasks and held weekly meeting.

**Comment by PinkSylvie** [comment]
This week I started researching with the class material on adding other patterns like composite or transactions into the documentation.

**Comment by FrancesC8** [comment]
Not much progress, will be working on first task for first sprint.

**Comment by ilikecheese117** [comment]
Made drafts for issues that will be used by the end of the sprint.

**Comment by ebrain-mendoza121** [comment]
Attended meetings, and reviewed possible task for first sprint.

**Comment by RicardoJRSs** [comment]
Working on tasks for first sprint

**Comment by sandy-marrero** [comment]
- Started work on LTT #132, preparing it for review this weekend.

**Comment by JadiellMV** [comment]
This week I fixed the JWT to make them more secure and started working on my LTTs.

**Comment by MrL9000** [comment]
Chose the tasks I'll be working on and met with some of the students of my team and one of the managers to get everything together all neat and tidy, will start working on my part this weekend.

**Reply by AngelCIICMorales** [reply]
Are you from our class?

**Comment by Gaby-Concepcion** [comment]
Begun outlining draw dominoes game mode from an implicit concept to an explicit concept as part of a LTT.

**Comment by rukitoyordan** [comment]
Working on task for sprint and LTTs.

**Comment by print-Eruki** [comment]
Working on tasks for this sprint

**Comment by Negroni1** [comment]
Reviewed class material for possible lecture topic task and drafted tasks for the milestone

**Comment by Joaquin939** [comment]
Completed a task for this sprint

**Comment by Jvstinoo** [comment]
Working on completing a task for this sprint.

**Comment by G4vp** [comment]
Working on completing a task for this sprint.

**Comment by AnthonyMed13** [comment]
Didn't have much time to work on the project but did a lecture topic documentation task

**Comment by frances-sepulveda** [comment]
Worked with Lecture Topic Tasks
